gateway/src/app.ts

```diff
@@ -42,6 +43,37 @@ app.get('/', (_req: Request, res: Response) => {
   res.status(200).json({ status: 'ok' });
 });
 
+app.get('/status', async (_req: Request, res: Response) => {
+  const avalanche = AvalancheRoutes.avalanche;
+  const ethereum = EthereumRoutes.ethereum;
```

Comment:
These would create instances if they do not exist before. Doing that isn't necessary and also makes it impossible to know networks Gateway wasn't connected to initially. I think what we need to do is create new static function(lets say `connected()`) in `Avalanche` and `Ethereum` classes which returns a boolean if `_instance` is set or not.

Reply:
**These would create instances if they do not exist before.** Not really, actually just the fact of importing the file will make it run and it will create the instance.
**I think what we need to do is create new static function** We could create such a function but it wouldn't be enough to know if the instance is set cause it could be misconfigured. In order to verify connection, we would still have to ask for the current block, so it would end up being a "connected" function that calls another "getCurrentBlock" function without adding that much value.
Something like 
```
async function connected() { 
try {
   await this.getCurrentBlockNumber()
   return true
} catch (err)
   return false
}
```
It's ok.

Reply:
I think we can further make it sophisticated like this:
```
public static async connected() { 
try {
   if (Ethereum._instance) {
      return await this.getCurrentBlockNumber() > 0 ? true : false;
   }
   return false
} catch (err) {
   return false
}
```

Reply:
but I was initially thinking something as simple as 
```
public static connected() {
    return Ethereum._instance ? true : false;
}
```

then in the return dict, we can have something like
```
{
...,
connected: connector.connected(),
currentBlockNumber: connector.connected() ? await connector.getCurrentBlockNumber() : -1
}
```